testdata/book/http_with_use_trace.yml

```
@@ -0,0 +1,40 @@
+desc: Test using HTTP with use Trace
+runners:
+  req:
+    endpoint: ${HTTPBIN_END_POINT:-https://httpbin.org/}
+    openapi3: https://tryapisproxy.com/spec/httpbin
+    useTrace: true
```

```suggestion
    trace: true
```

Since this parameter is to enable, not to use, tracing, `trace:` may be sufficient like `debug:`

---

Now that you mention it, is the useCookie the same?

---

I think `useCookie:` is ok.
Because, "Cookie" is always enabled on the HTTP protocol and runn using its cookie feature.

---

However, I have no objection to changing it to `cookie:`.